fio/lib/README.md

```diff
@@ -0,0 +1,54 @@
+# FIOWallet Class
+
+The FIOWallet class provides functionalities for interacting with the FIO blockchain, including mapping a public address to a FIO address.
+
+## Installation
+
+To use the FIOWallet class in your project, you can install it via npm:
+
+```bash
+npm install fio-wallet
```

Comment:
The package isn't published. Update these instructions. In general, someone going through the README should be able to use the package without any issues.

Reply:
Done